packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -144,7 +148,30 @@ class TerraService {
    */
   async onComplete(_, config) {
     try {
-      if (this.serviceOptions.useRemoteReferenceScreenshots && !this.serviceOptions.buildBranch.match(BUILD_BRANCH.pullRequest) && this.serviceOptions.buildType === BUILD_TYPE.branchEventCause) {
+      if (this.serviceOptions.useRemoteReferenceScreenshots && process.env.SCREENSHOT_MISMATCH_CHECK && this.serviceOptions.buildBranch.match(BUILD_BRANCH.pullRequest)) {
+        if (!this.serviceOptions.gitToken || !this.serviceOptions.gitApiUrl) {
+          throw new Error('No git token recieved');
+        }
+
+        const packageJson = fs.readJsonSync(path.join(process.cwd(), 'package.json'));
+        const repoUrl = new URL(packageJson.repository.url);
+        const repoName = repoUrl.pathname.match(/[^/]+/g);
+        const octokit = new Octokit({ baseUrl: `${this.serviceOptions.gitApiUrl}`, auth: `${this.serviceOptions.gitToken}` });
+        const message = `:warning: :bangbang: **WDIO MISMATCH** \n\nCheck that screenshot change is intended at: ${this.serviceOptions.buildUrl}`;
```

Comment:
This is what the Consumption Guide says about this alert at the end:

> Pull Request Build - Intentional changes in a project can alter wdio screenshots. Like the branch build, a pull request build will also download the reference screenshots from Nexus to use for comparisons. Unlike the branch build, should screenshot mistmaches occur the pull request build will not fail because the mismatches are expected due to the intentional changes in the pull request. When there are screenshot mismatches, an alert will be displayed in the pull request and to suggest to review the mismatches to verify the mismatches are expected. Latest sccreenshots generated by this type of build will also not be uploaded to Nexus.

Do we need to update the guide to provide better directions/steps to highlight what they need to do? Or perhaps add additional guidance on this message so they know what the expectations are.  
A consumer reading this message probably won't know how to proceed after reviewing the mismatch screenshots.

Reply:
Thinking on it I think a notice of what will happen if mismatches are still there upon merge is needed in the message. I updated the message to as follows but expect we will want to properly word this so I'm open to suggestions.

> :warning: :bangbang: **WDIO MISMATCH** 
> 
> If screenshot change is intended, remote reference screenshots will be updated upon PR merge.
> 
> Check that screenshot change is intended at: {buildURL}

https://github.com/cerner/terra-toolkit/pull/755/commits/627a4865cc4c1e83b773cfe1799d89d092b314b2#diff-a6ba3c1d16c08e58e7ee3994f3ab541b63b2a3a8dafb0de9daed2c8090c948d9R160

Reply:
Does the buildURL link to the build or to the build artifacts showing the screenshot differences?

Do we think it is more logical to swap the order of these two sentences? It seems more logical to me that we should first tell where to look and then explain what will happen. 

For example:
```
Check that screenshot change is intended at: ${this.serviceOptions.buildUrl}
If screenshot change is intended, remote reference screenshots will be updated upon PR merge.
```

Should we also add something about what to do if the screenshots are not intended after reviewing them and emphasize that they should carefully review the mismatches. Normally when a PR has a successful build, it is easy for consumers to think that everything is good to go. But in this case, they should closely review any unintended mismatches and get those corrected. So I think it's important to call out what to do when there are unintended mismatches.

I'm curious to hear what other people think.

Reply:
buildURL will link right to the specific job main page of that branch.

Yea that sounds good to me so how about

> Check that screenshot change is intended at: ${this.serviceOptions.buildUrl}
> If screenshot change is intended, remote reference screenshots will be updated upon PR merge.
> If screenshot change is untended, please fix screenshot issues before PR merge to prevent them from being uploaded.

I'm also curious what people think about the 1 comment per PR thing. Because I am worried about what happens when they get that message early, make a bunch of changes, and a new issue in screenshots is added that they don't catch because the message only posts on the first time a build would fail due to screenshot mismatch? I get we don't want to spam comment on a PR so I definitely think we need to mention we're only posting once per PR with this message whether that be in the message we post or in the documentation for updating.

Reply:
New message here https://github.com/cerner/terra-toolkit/pull/755/commits/b157573530a80aae84b5707f582dac14aa87472a#diff-a6ba3c1d16c08e58e7ee3994f3ab541b63b2a3a8dafb0de9daed2c8090c948d9R160